src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -466,6 +466,7 @@ public function formAction($id, Request $request)
                 if ($form->isValid()) {
                     //define POST values for keeping legacy adminController skills
                     $_POST = $modelMapper->getModelData($formData, $isMultiShopContext) + $_POST;
+                    $_POST['form'] = $formData;
```

Comment:
This is very interesting line, is it related to all the problems with not working product saving in some scenarios? How did you catch it?

Reply:
It's maybe related to some scenarios, but not all of them :smile: 
By editing the ProductInformation Form, I found that my changes were not catch in the legacy controller (don't understand why they not fully migrate this page :/) and step by step I realized that the `$_POST` needs to be filled.